src/templates/program-page.js

```diff
@@ -259,224 +216,170 @@ function renderProgramInfo (courseData, courseNotes, variantDataHeading, variant
+        { /**** Header and Title ****/ }
+        <div className={!contentExists(heroImage) && !contentExists(videoData) ? "no-thumb" : null} id="rotator">
+            {videoData ?
+            <HeroVideo videoURL={videoData.field_media_oembed_video} videoWidth={videoData.field_video_width} videoHeight={videoData.field_video_height} videoTranscript={videoData.relationships.field_media_file?.localFile.publicURL} />
             :
             <Hero imgData={heroImage} />
-        }
-        <div className="container ft-container">
-          <h1 className="fancy-title">{title}</h1>
+            }
+            <div className="container ft-container"><h1 className="fancy-title">{title}</h1></div>
         </div>
-      </div>
 
-      { /**** Tags and Call to Action Button ****/ }
-      <div className="full-width-container bg-dark">
-          <div className="container">
-              <section className="row row-with-vspace site-content">
-                  <div className="col-md-9 content-area">
-                    {tagData && tagData.length > 0 ?  
-                      (<Tags tagData={tagData} />)
-                      : null
-                    }   
-                  </div>
-                  <div className="col-md-3">
-                    {callToActionData.map((cta, index) => (
-                      <CallToAction key={index} href={cta.node.field_call_to_action_link.uri} 
-                        goalEventCategory={contentExists(cta.node.relationships.field_call_to_action_goal)? cta.node.relationships.field_call_to_action_goal.name: ``} 
-                        goalEventAction={contentExists(cta.node.relationships.field_call_to_action_goal)? cta.node.relationships.field_call_to_action_goal.field_goal_action: ``} 
-                        classNames='btn btn-uogRed apply' >
-                        {cta.node.field_call_to_action_link.title}
+        { /**** Blurb and Call to Action Button ****/ }
+        {ogDescription || contentExists(callToActionData) ? 
+        <div className="full-width-container bg-dark">
+            <div className="container">
+                <section className="row mx-2">
+                    {ogDescription && <div className="col-md-9"><p className="fs-2">{ogDescription}</p></div>}
+                    {contentExists(callToActionData) && 
+                    <div className="col-md-3">
+                      <CallToAction href={callToActionData[0]?.node.field_call_to_action_link.uri} 
+                        goalEventCategory={callToActionData[0]?.node.relationships.field_call_to_action_goal?.name} 
+                        goalEventAction={callToActionData[0]?.node.relationships.field_call_to_action_goal?.field_goal_action}>
+                        {callToActionData[0]?.node.field_call_to_action_link.title}
```

Comment:
Recommend re-adding the classNames as a parameter for CallToAction at this level rather than hard-coding the look and feel inside the callToAction widget.